.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,17 @@
+# https://github.com/marketplace/actions/auto-update
```

Comment:
If a PR has auto-merge enabled and includes either a `core-team` or `devrel` tag, auto-update.